client/my-sites/comments/comment-list/comment-list-header.jsx

```diff
@@ -47,7 +47,9 @@ export const CommentListHeader = ( {
 				actionText={ translate( 'View Post' ) }
 				backHref={ `/comments/all/${ siteSlug }` }
 			>
-				<div className="comment-list__header-title">{ postTitle }</div>
+				<div className="comment-list__header-title">
+					{ postTitle.trim() || translate( 'Untitled' ) }
```

Comment:
Any reason not to update postTitle in mapStateToProps?

Reply:
None whatsoever, just wanted to be symmetrical with the other file. 🙂